test/users.test.js

```diff
@@ -0,0 +1,93 @@
+const request = require('supertest');
+const truncateDatabase = require('./setup');
+const app = require('../app');
+
+beforeAll(async () => {
+  process.env.NODE_ENV = 'test';
```

Comment:
move `process.env.NODE_ENV` to the config file.

Reply:
Done :)